requirements.txt

```diff
@@ -1,5 +1,5 @@
-pause==0.1.2
-python-dateutil==2.7.3
-selenium==3.14.0
-six==1.11.0
-urllib3==1.24.2
+pause>=0.1.2
```

Comment:
I'm not a big fan of solely using >= for versions. It's only a matter of time before one of these has a new release with a breaking change. If the latest version for all these works, then let's just cap it at whatever that major version is for now (e.g. pause>=0.1.2,<1.0).

Reply:
I see your point. I just changed it back to what it was, as this seems like a matter of preference, and you made your original choice for a reason.